Documentation/DocuBlocks/Rest/Cursors/post_api_cursor.md

```diff
@@ -6,6 +6,23 @@
 
 A JSON object describing the query and query parameters.
 
+@RESTHEADERPARAM{x-arango-allow-dirty-read,string,optional}
+If the "x-arango-allow-dirty-read" header is given, then it must contain
+the value `true`. This will allow the coordinator to ask any shard
+replica for the data and not just the leader. However, this may then
+produce "dirty reads", see
+[Dirty Reads](./document-address-and-etag.html#dirty-reads)
```

Comment:
Internal references do not work in Swagger UI, but they (mostly) will in the docs. We can substitute with absolute URLs like done here:
https://github.com/arangodb/arangodb/blob/733402d122524aa1f42f76b9bbef062e4191b836/Documentation/DocuBlocks/Rest/Views/post_api_view_arangosearch.md?plain=1#L16
This has the disadvantage of linking to the latest stable version of the docs, which may not match the version in use and it also changes over time. Alternatively, we can remove the links entirely and leave it to the users to find related information.